content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,133 @@
+les caractéristiques sexuelles, l'identité et l'expression de genre, 
+le niveau d'expérience, l'éducation, le statut socio-économique, 
+la nationalité, l'apparence personnelle, la race, la religion, 
+ou l'identité et l'orientation sexuelle.
+
+Nous nous engageons à agir et interagir de manière à contribuer à une communauté 
+ouverte, accueillante, diversifiée, inclusive et saine.
+
+## Nos critères
+
+Exemples de comportements qui contribuent à créer un environnement positif :
+
+* Faire preuve d'empathie et de bienveillance envers les autres
+* Être respectueux des opinions, points de vue et expériences divergents
+* Donner et recevoir avec grâce les critiques constructives
+* Assumer ses responsabilités et s'excuser auprès des personnes affectées par nos erreurs et apprendre de ces expériences
+* Se concentrer sur ce qui est le meilleur non pas uniquement pour nous en tant qu'individu, mais aussi pour l'ensemble de la communauté
+
+Exemples de comportements inacceptables :
+
+* L'utilisation de langage ou d'images sexualisés et d'attentions ou d'avances sexuelles de toute nature
+* Le _trolling_, les commentaires insultants ou désobligeants et les attaques
+  personnelles ou d'ordre politique
+* Le harcèlement en public ou en privé
+* La publication d'informations privées de tierces personnes, telles que des
+  adresse postale ou une adresse électronique, sans leur autorisation explicite
+* Toute autre conduite qui pourrait raisonnablement être considérée comme inappropriée
+  dans un cadre professionnel
+
+## Nos responsabilités
+
+Les dirigeant•e•s de la communauté sont chargé•e•s de clarifier et de faire respecter nos normes de
+comportements acceptables et prendront des mesures correctives appropriées et équitables en
+réponse à tout comportement qu'ils ou elles jugent inapproprié, menaçant, offensant ou nuisible.
+
+Les dirigeant•e•s de la communauté ont le droit et la responsabilité de supprimer, modifier ou rejeter
+les commentaires, les contributions, le code, les modifications de wikis, les rapports d'incidents ou de bogues et autres contributions qui
+ne sont pas alignés sur ce code de conduite, et communiqueront les raisons des décisions de modération
+le cas échéant.
+
+## Portée d'application
+
+Ce code de conduite s'applique à la fois au sein des espaces du projet ainsi que
+dans les espaces publics lorsqu'un individu représente officiellement le projet ou sa
+communauté. Font parties des exemples de représentation d'un projet ou d'une
+communauté l'utilisation d'une adresse électronique officielle, la publication sur
+les réseaux sociaux à l'aide d'un compte officiel ou le fait d'agir en tant que représentant•e désigné•e
+lors d'un événement en ligne ou hors-ligne
+
+## Application
+
+Les cas de comportements abusifs, harcelants ou tout autre comportement
+inacceptables peuvent être signalés aux dirigeant•e•s de la communauté responsables de l'application à

```

Comment:
I think we shouldn't restraint it to an "application" and use "projet".
```suggestion
inacceptables peuvent être signalés aux dirigeant•e•s de la communauté responsables du projet à
```

Reply:
```suggestion
inacceptables peuvent être signalés aux dirigeant•e•s de la communauté responsables de l'application du code de conduite à
```

Reply:
It's about enforcing the code of conduct ("l'application du code de conduite") not apps.

Reply:
Ok, seems I didn't got it righ then !